### PR TITLE
Feat/implement slack api

### PR DESCRIPTION
**Pull request by AnsMann**

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -97,6 +98,16 @@ export default function App() {
             </>
           )}
         />
+        <Route
+          exact
+          path="/cards/:id/share"
+          render={props => (
+            <>
+              <Header title={'Share via slack'} />
```
Comment (L-M-K-B) [inline src/app/App.js]: Implement Header in ShareDiaryEntry Page?

File: src/app/ShareDiaryEntry.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { findIndex, handleSlackContactData } from './utils'
+import { getContacts, sendMessage } from './services'
+import { SlackContactList } from './SlackContactList'
+
+const ShareContainer = styled.section`
```
Comment (L-M-K-B) [inline src/app/ShareDiaryEntry.js:13]: Sort in alphabetical order?

File: src/app/ShareDiaryEntry.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  }
+`
+
+const SearchArea = styled.section`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  height: 200px;
```
Comment (L-M-K-B) [inline src/app/ShareDiaryEntry.js]: Needs to adjusted afterwards.

File: src/app/ShareDiaryEntry.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  position: relative;
+`
+const ResultArea = styled.section`
+  padding: 15px;
+  height: 400px;
```
Comment (L-M-K-B) [inline src/app/ShareDiaryEntry.js]: See comment above: Needs to adjusted afterwards.

File: src/app/ShareDiaryEntry.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+const Line = styled.div`
+  position: relative;
+  left: 10%;
+  width: 80%;
+  border-bottom: solid 1px #007fbf;
+  margin-bottom: 10px;
+`
+
+export function ShareDiaryEntry({ match, diaryEntries }) {
```
Comment (L-M-K-B) [inline src/app/ShareDiaryEntry.js]: Unpack id from match in app.js?

File: src/app/ShareDiaryEntry.js [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  const [searchInput, setSearchInput] = useState('')
+  const [slackContacts, setSlackContacts] = useState([])
```
Comment (L-M-K-B) [inline src/app/ShareDiaryEntry.js]: Maybe set results in local storage in order to get a visible list of contacts faster.

**Review by L-M-K-B — APPROVED** [review]
Das hast du wundervoll gemacht.

File: src/app/ShareDiaryEntry.js (hunk 2 above)
Comment (AnsMann) [reply src/app/ShareDiaryEntry.js:13]: done :)

File: src/app/ShareDiaryEntry.js (hunk 3 above)
Comment (AnsMann) [reply src/app/ShareDiaryEntry.js]: you are right. i have that in mind

File: src/app/ShareDiaryEntry.js (hunk 5 above)
Comment (AnsMann) [reply src/app/ShareDiaryEntry.js]: i corrected this one

File: src/app/ShareDiaryEntry.js (hunk 6 above)
Comment (AnsMann) [reply src/app/ShareDiaryEntry.js]: implement that too

File: src/app/App.js (hunk 1 above)
Comment (AnsMann) [reply src/app/App.js]: thanks. i changed this

**Pull request merged**